website/src/content/deployment-examples/commercetools-connect.mdx

```diff
@@ -0,0 +1,119 @@
+
+# Create your application
+
+1. Install a Connect [starter template](https://docs.commercetools.com/connect/development#install-a-starter-template).
+2. Open the `mc-app` folder in the Connect starter template and [install the Custom Applications starter template](/getting-started#installing-a-starter-template) with the name `sample-custom-application` inside this folder.
+3. Add your environment variables to `custom-application-config.mjs`.
+
+    ```js title="custom-application-config.mjs" highlightLines="5-13"
+    const config = {
+      name: 'Example Custom Application',
+      entryPointUriPath,
+      cloudIdentifier: '${env:CLOUD_IDENTIFIER}',
+      env: {
+        development: {
+          initialProjectKey: '${env:CTP_PROJECT_KEY}',
+        },
+        production: {
+          applicationId: '${env:CUSTOM_APPLICATION_ID}',
+          url: '${env:APPLICATION_URL}',
+        },
+      },
+      oAuthScopes: {
+        view: ['view_products'],
+        manage: ['manage_products'],
+      },
+      icon: '${path:@commercetools-frontend/assets/application-icons/rocket.svg}',
+      mainMenuLink: {
+        defaultLabel: 'Template starter',
+        labelAllLocales: [],
+        permissions: [PERMISSIONS.View],
+      },
+      submenuLinks: [
+        {
+          uriPath: 'channels',
+          defaultLabel: 'Channels',
+          labelAllLocales: [],
+          permissions: [PERMISSIONS.View],
+        },
+      ],
+    };
+
+    export default config;
+    ```
+
+4. Open `connect.yaml` in the root of the Connect starter template and replace it with the following. The environment variables you included in `custom-application-config.mjs` should be defined here. Sensitive variables, such as API client information, should be defined within `securedConfiguration`.
+
+      ```yml
+      deployAs:
+        - name: example-custom-application
+          applicationType: mc-app
+          endpoint: /
+          configuration:
+            standardConfiguration:
+              - key: CTP_PROJECT_KEY
+                description: project key from the Composable Commerce Project
+                default: "your-project-key"
+              - key: CTP_REGION
+                description: region of the Composable Commerce Project
+                required: true
+              - key: CUSTOM_APPLICATION_ID
+                description: the Custom Application ID
+                required: true
+              - key: CLOUD_IDENTIFIER
+                description: the cloud identifier
+                default: "gcp-eu"
+            securedConfiguration:
+              - key: CTP_CLIENT_ID
+                description: client id from the Composable Commerce Project
+                required: true
+              - key: CTP_CLIENT_SECRET
+                description: client secret from the Composable Commerce Project
+      ```
```

Comment:
> Sensitive variables, such as API client information, should be defined within `securedConfiguration`

Nit: a Custom Application doesn't require sensitive variables. Mentioning it here is a bit confusing. I would prefer to only mention the relevant information related to Custom Applications. The `securedConfiguration` part can be omitted.

Reply:
Thanks for the feedback, I'll edit this to include only `CUSTOM_APPLICATION_ID` and `CLOUD_IDENTIFIER`.